ownCloud/Theming/NSObject+ThemeApplication.swift

```diff
@@ -271,3 +291,48 @@ extension UITableViewController : ThemeableSectionHeader, ThemeableSectionFooter
 		}
 	}
 }
+
+extension UISearchBar {
+
+	func getTextField() -> UITextField? {
+			return value(forKey: "searchField") as? UITextField
```

Comment:
Please use `UISearchBar.searchTextField` on iOS 13+ and only use this lack-of-public-API workaround for iOS 12 and older.

Reply:
Fixed